tests/unit/test_context7_client_extended.py

```
@@ -262,7 +264,7 @@ def test():
 ```
 '''
         
-        result = client._parse_docs_content(content)
+        result = client._parse_docs_content(content)  # pyright: ignore[reportArgumentType]
```

Pylint (3.3.7)

[warning] Access to a protected member _parse_docs_content of a client class

(W0212)